osquery/main/fuzz.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/config.h>
+#include <osquery/database.h>
+#include <osquery/registry.h>
+#include <osquery/sql.h>
+
+void init() {
+  osquery::registryAndPluginInit();
+  osquery::DatabasePlugin::setAllowOpen(true);
+  osquery::Registry::get().setActive("database", "ephemeral");
+  osquery::DatabasePlugin::initPlugin().ok();
+
+  osquery::PluginRequest r;
+  r["action"] = "detach";
+  r["table"] = "file";
+
+  osquery::PluginResponse rsp;
+  osquery::Registry::get().call("sql", r, rsp);
+}
+
+#if 0
```

Comment:
Can we have a set of macros at the top that will turn on different fuzz features?

Reply:
This was meant as example code. Not really for switching things off and on, but happy to change if you have strong opinions.